frontend/src/features/election_management/components/report/ElectionReportPage.tsx

```diff
@@ -14,17 +14,22 @@ import {
   COMMITTEE_SESSION_STATUS_CHANGE_REQUEST_PATH,
 } from "@/types/generated/openapi";
 import { committeeSessionLabel } from "@/utils/committeeSession";
-import { formatFullDateWithoutTimezone } from "@/utils/format";
+import { formatFullDateWithoutTimezone } from "@/utils/dateTime";
 
-import { directDownload } from "../utils/download";
-import cls from "./ElectionManagement.module.css";
+import { directDownload } from "../../utils/download";
+import cls from "../ElectionManagement.module.css";
 
 export function ElectionReportPage() {
   const { committeeSession, election } = useElection();
   const client = useApiClient();
   const navigate = useNavigate();
   const [changeStatusError, setChangeStatusError] = useState<AnyApiError | null>(null);
 
+  // Redirect to update details page if committee session details have not been filled in
+  if (committeeSession.location === "" || committeeSession.start_date === "" || committeeSession.start_time === "") {
```

Comment:
Could `start_date` also be `undefined` or `null`?
Maybe we should handle these cases in `formatFullDateWithoutTimezone`, like returning '-' if there was an error parsing the date.

Currently it wil return the string "Invalid Date" in my browser:

```ts
(new Date('tijd voor frietjes')).toLocaleTimeString("nl-NL", {
    hour: "numeric",
    minute: "numeric",
    day: "numeric",
    month: "long",
    year: "numeric",
  });
  ```